Common/cpp/ReanimatedRuntime/ReanimatedRuntime.cpp

```diff
@@ -20,6 +20,7 @@ using namespace facebook;
 using namespace react;
 
 std::shared_ptr<jsi::Runtime> ReanimatedRuntime::make(
+    jsi::Runtime *reactRuntime,
```

Comment:
#3722 introduces `JSRuntimeHelper` which holds `jsi::Runtime *uiRuntime_;` and `jsi::Runtime *rnRuntime_;`. We could unify this PR with the new naming convention and use `rnRuntime` here as well. 
```suggestion
    jsi::Runtime *rnRuntime,
```